presto-hive/src/test/java/com/facebook/presto/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -262,7 +263,8 @@ private static BackgroundHiveSplitLoader backgroundHiveSplitLoader(
                 EXECUTOR,
                 2,
                 false,
-                false);
+                false,
+                null);
```

Comment:
Just a reminder this would also need to be changed to `Optional.empty()` if we use `Optional` in the field.

Reply:
yes sure